test/v2/client/test_client_setup.go

```diff
@@ -3,14 +3,13 @@ package client
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/Layr-Labs/eigenda/common"
+	"github.com/Layr-Labs/eigensdk-go/logging"
```

Comment:
nit: Does your IDE use a different import ordering scheme?

Reply:
I think it's just ordering them alphabetically. Is there a different schema we should be using?

Reply:
https://pkg.go.dev/golang.org/x/tools/cmd/goimports

Reply:
My IDE uses the `gofmt` standards, I'm pretty sure. The difference here is that I deleted the newline between `"testing"` and `"github.com/Layr-Labs/eigenda/common"`, which caused them to be ordered alphabetically with everything else. If we want to keep github dependencies in a separate block, I'm fairly sure it requires hand maintenance.

Reply:
reformatted